pipeline/metadata/schema.py

```diff
@@ -93,6 +53,49 @@ def merge_ip_metadata(base: IpMetadata, new: IpMetadataWithKeys) -> None:
     base.name = new.name
 
 
+@dataclass
+class SatelliteTags():
+  """Satellite tags that are not metadata specifically about the answer ip"""
+  http: Optional[str] = None
+  cert: Optional[str] = None
+  matches_control: Optional[str] = None
+
+
+@dataclass
+class SatelliteAnswer():
+  """Class for keeping track of Satellite answer content"""
+  # Keys
+  ip: str
+  tags: SatelliteTags = dataclasses.field(default_factory=SatelliteTags)
```

Comment:
This "tags" concept is a bit arbitrary and is mixing things we observed (http_body, cert), with things we derive (matches_control)
Can you keep those separate?
I'd just keep them inline here for now, rather than introducing a concept that is not cleaar.

Reply:
done